src/components/Modal.jsx

```diff
@@ -195,6 +248,8 @@ Modal.defaultProps = {
   title: 'Please give me a title :)',
   content: 'Please give me some content :)',
   image: '',
+  sound: '',
```

Comment:
@patrickocoffeyo so in the case that this has a sound, the value for this property would be the sound file. For example it would be defined in the modal like so:
```
<Modal
    id="modal__upstairs"
    title="Upstairs Playroom"
    position={{ x: -26.15, y: 5.70, z: -10.41 }}
    image={require('../../assets/images/jpg/upstairs.jpg')}
    sound={require('../../assets/sounds/upstairs.mp3')}
    volume={5}
 />
```
Thoughts?